### PR TITLE
ENH: add force_2d and force_3d

### PR DESCRIPTION
**Pull request by martinfleis**
xref #2010

### CODE REVIEW

File: geopandas/tests/test_geom_methods.py [hunk 1]
```diff
@@ -1731,3 +1731,46 @@ def test_offset_curve(self):
...
+            ],
+            crs=4326,
+        )
+        assert_geoseries_equal(expected, self.landmarks_mixed_empty.force_2d())
+
+    def test_force_3d(self):
+        expected = GeoSeries(
+            [
+                Point(-73.9847, 40.7484, 30.3244),
+                Point(-74.0446, 40.6893, 31.2344),
```
Comment (m-richards) [inline geopandas/tests/test_geom_methods.py:1751]:
```suggestion
                self.esb,
                self.sol,
```

File: geopandas/tests/test_geom_methods.py [hunk 2]
```diff
@@ -1731,3 +1731,46 @@ def test_offset_curve(self):
...
+                self.pt_empty,
+            ],
+            crs=4326,
+        )
+        assert_geoseries_equal(expected, self.landmarks_mixed_empty.force_3d())
+
+        expected = GeoSeries(
+            [
+                Point(-73.9847, 40.7484, 30.3244),
+                Point(-74.0446, 40.6893, 31.2344),
```
Comment (m-richards) [inline geopandas/tests/test_geom_methods.py:1762]:
```suggestion
                self.esb,
                self.sol,
```

**Review by m-richards — APPROVED** [review]
Thanks Martin!

**Review by jorisvandenbossche — APPROVED** [review]

**Comment by jorisvandenbossche** [comment]
Thanks!

**Pull request merged**